### ISSUE TITLE
Redhat/Fedora: cannot be run because of missing libpixbufloader-png.so

### ISSUE THREAD

**Issue by LicoMonch** [rimpy-custom/RimPy#38]
Fedora/Redhat cannot simply run RimPy, because it crashes when you click on one of the folder select buttons:
```
(RimPy:38277): Gtk-WARNING **: 02:28:30.668: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Adwaita/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Adwaita/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Aborted (core dumped)
```
That's a known issue: https://bugzilla.redhat.com/show_bug.cgi?id=2031223

**Comment by rimpy-custom** [comment]
May you test with new RimPy pre-release?

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
